networkstatsquerier/fetcher/defaultlist.go

```diff
@@ -0,0 +1,55 @@
+package fetcher
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// DefaultList implements the Endpoints interface for endpoint lists in the default format
+// which is the format that matches the ListOfEndpoints struct
+type DefaultList struct{}
```

Comment:
change to epic list.
perhaps also have a default list, but have the default list take a "source" as an argument.